extensions/cornerstone/src/utils/measurementServiceMappings/RectangleROI.ts

```diff
@@ -13,8 +15,10 @@ const RectangleROI = {
     getValueTypeFromToolType,
     customizationService
   ) => {
-    const { annotation, viewportId } = csToolsEventDetail;
+    const { annotation } = csToolsEventDetail;
```

Comment:
It is a code smell that a modification to an external library has the same code modified in the same way across multiple files.  Generally that means that a super class should be created that has most of the shared code, with only the small/specific bit which is needed to be changed modified.  That would also make it much easier to create new annotation types.  I'm not asking for that to be done now, but please have an issue for it.